lib/mas/cms/entity/static_page.rb

```diff
@@ -2,8 +2,8 @@ module Mas::Cms
   class StaticPage < Entity
     Alternate = Struct.new(:title, :url, :hreflang)
 
-    attr_accessor :type, :title, :description, :body, :label, :meta_description,
-                  :translations
+    attr_accessor :type, :title, :description, :meta_title,
+                  :body, :label, :meta_description, :translations
```

Comment:
Maybe unrelated to this PR: did you get what's the difference between `description` and `meta_description`? Asking because the latter seems to be just a fallback when the former is absent: 

https://github.com/moneyadviceservice/mas-cms-client/blob/892ed44377367a4b11a8d1514a82428a99b9de3c/lib/mas/cms/entity/static_page.rb#L14-L16

Reply:
It seems like a variant between `Articles` and `Documents`. My best guess is that one was legacy and has been kept for retro-compatibility. Might be worth looking into refactoring, but I would say outside fo the scope of this PR